lib/modules/txt.mjs

```diff
@@ -454,3 +454,26 @@ export async function validateCopyrightDate (doc, { mode = MODES.NORMAL, year }
 
   return result
 }
+
+/**
+ * Validate document name
+ *
+ * @param {Object} doc Document to validate
+ * @param {Object} [opts] Additional options
+ * @param {number} [opts.mode=0] Validation mode to use
+ * @returns {Array} List of errors/warnings/comments or empty if fully valid
+ */
+export async function validateDocumentName (doc, { mode = MODES.NORMAL } = {}) {
+  const result = []
+
+  if (!doc.data.title) {
```

Comment:
You are checking the document title. But task says to check document NAME, that usually looks like draft-ietf-idr-rt-derived-community-05. Please fix and improve parser so it could work in different ways, when there is no title and only docname, vice versa and etc.